tests/bdd/steps/__init__.py

```diff
@@ -0,0 +1,4 @@
+# As this class currently defines no steps it isn't automatically loaded
```

Comment:
It took me two passes to grok that _this class_ refers to `HAPIContext`, but maybe I'm just slow on the uptake.

Reply:
I'll reword this to be less confusing.